src/constants.ts

```diff
@@ -337,14 +337,6 @@ export const QUESTIONAIRRE: inquirer.QuestionCollection = [
     when: (answers) => ['remote', 'both'].includes(answers.backend)
   },
 
-  // ANONYMOUS METRIC COLLECTION
-  {
-    type: 'confirm',
-    name: 'allowAnonymousMetrics',
-    message: 'Allow Nightwatch to collect completely anonymous usage metrics?',
-    default: false
-  },
-
```

Comment:
No need to remove this. Since we are already passing an answer to this question while calling `prompt`, this question will be automatically skipped.